astropy/nddata/tests/test_utils.py

```diff
@@ -465,3 +487,16 @@ def test_naxis_update(self):
         c = Cutout2D(self.data, self.position, (ysize, xsize), wcs=self.wcs)
         assert c.wcs._naxis[0] == xsize
         assert c.wcs._naxis[1] == ysize
+
+    def test_crpix_maps_to_crval(self):
+        w = Cutout2D(self.data, (0, 0), (3, 3), wcs=self.sipwcs,
+                     mode='partial').wcs
+        pscale = np.sqrt(proj_plane_pixel_area(w))
+        assert_allclose(
+            w.wcs_pix2world(*w.wcs.crpix, 1), w.wcs.crval,
```

Comment:
since this is not python2.7 or python 3.4 compatible, so I'll only backport to the 3.0.x bugfix branch

Reply:
@bsipocz I am not sure I understand all of this. What is not compatible with Python 2.7? Is there anything I can do?